docs/source/developer_guide/pipelines.md

```diff
@@ -45,7 +45,7 @@ For more details on the **pipeline json definition** see it's [json schema](http
 				"label": "generate-contributions",
 				"component_parameters": {
 					"filename": "demo-pipelines/generate-contributions.ipynb",
-					"runtime_image": "tensorflow/tensorflow:2.0.0-py3",
+					"runtime_image": "tensorflow/tensorflow:7c01f75d58fadc2cd1109d5baac1925ed131e05925d840b1b49363c794d1c4db",
```

Comment:
Maybe we could add a link to the guide about how and where to find those hashes on the dockerhub website for a given OS Environment linux/amd64 e.g. [https://hub.docker.com/r/tensorflow/tensorflow and ](https://hub.docker.com/r/tensorflow/tensorflow/tags?page=1&name=2.0.0-py3). But in the case of that image, the hash is [0b236338fac6c3361cf3ae1448f8c053994e260c1edc4fa63ed80adb3045abb2](https://hub.docker.com/layers/tensorflow/tensorflow/2.0.0-py3/images/sha256-0b236338fac6c3361cf3ae1448f8c053994e260c1edc4fa63ed80adb3045abb2?context=explore)